johnsonandjohnson/Bodiless-JS
# docs(bodiless-documentation): lint error on navigiation #1156

## Changes
Remove the component names from TOC that was causing weird symlink thing to happen that then caused lint to fail.
They are not needed to be in TOC,

## Test Instructions

- npm run setup
- npm run start
- visit http://localhost:8005/___docs/#/Components and confirm you can see Youtube & Navigation doc
- npm run lint  and verify no lint error no longer happens

## Related Issues
https://github.com/johnsonandjohnson/Bodiless-JS/pull/1146
